app/models/like.rb

```diff
@@ -0,0 +1,15 @@
+class Like < ApplicationRecord
+  belongs_to :post, foreign_key: true, optional: true
+  belongs_to :user, foreign_key: true, optional: true
```

Comment:
- [ ] OPTIONAL 
(_However required for your code to work as intended_)
Going by the given _ERD_, establishing the _Like_ **association** should be such that _Post_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User`.

  Kindly ensure to improve upon this aspect.